test/gui/shared/steps/steps.py

```diff
@@ -536,22 +543,31 @@ def createPublicShare(context, resource, role):
     clickButton(waitForObject(radioObjectName))
     clickButton(waitForObject(names.oCC_ShareLinkWidget_createShareButton_QPushButton))
 
-@When('the user creates a new public link for folder "|any|" using the client-UI with these details:')
-def step(context, resource):
-    role = ''
-    for row in context.table:
-        if row[0] == 'role':
-            role=row[1]
-            break
+@When('the user creates a new public link for folder "|any|" using the client-UI with these details:')    
+def step(context, resource):    
+    role = ''    
+    for row in context.table:    
+        if row[0] == 'role':    
+            role=row[1]    
+            break    
 
-    if role == '':
-        raise Exception("No role has been found")
-    else:
-        createPublicShare(context, resource, role)
+    if role == '':    
+        raise Exception("No role has been found")    
+    else:    
+        createPublicShare(context, resource, role)    
 
 
-@When('the user creates a new public link for folder "|any|" with "|any|" using the client-UI')
-def step(context, resource, role):
-    createPublicShare(context, resource, role)
+@When('the user creates a new public link for folder "|any|" with "|any|" using the client-UI')    
+def step(context, resource, role):    
+    createPublicShare(context, resource, role)    
```

Comment:
same here

Reply:
same here, the IDE changed it.

Reply:
there are some formatting errors around the code, e.g. Tabs vs. Spaces or like here extra spaces at the end. Maybe you can make a reformatting PR first

Reply:
actually I see the IDE added extra spaces, that is weird

Reply:
@Talank good idea, please open an issue and place it in the backlog

Reply:
@Talank after https://github.com/owncloud/client/pull/8636 has been merged. You can now use Black and fix these indentations.